openstack/datadog_checks/openstack/openstack.py

```diff
@@ -663,16 +663,36 @@ def get_all_server_ids(self, filter_by_host=None):
 
         return server_ids
 
+    def get_project_name_from_id(self, tenant_id):
+        url = "{0}/{1}/{2}/{3}".format(self.keystone_server_url, DEFAULT_KEYSTONE_API_VERSION, "projects", tenant_id)
```

Comment:
nit - With Python 2.7 in this case there's no need to specify the numbers for the placeholders